modules/bbmap/align/main.nf

```diff
@@ -0,0 +1,59 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BBMAP_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::bbmap=38.92 bioconda::samtools=1.13 pigz=2.6" : null)
```

Comment:
I don't see any `pigz` command in the script block, is this a requirement of the `bbmap` wrapper script?

Reply:
Disregard this comment, I see from the test workflows this is indeed the case (leaving the comment though for future reference ;)).